website/docs/release.md

```diff
@@ -32,28 +32,106 @@ All releases follow the format `vX.Y.Z`, where X is the major version, Y is the
 
 ## Release Process
 
-Copacetic uses [GoReleaser](https://goreleaser.com/) for automating releases.
+Copacetic uses [GoReleaser](https://goreleaser.com/) for automating releases. The release process differs depending on whether you're cutting a major/minor release or a patch release.
 
-### Steps to Cut a Release
+### Major/Minor Release Process
+
+For major and minor releases (e.g., v0.11.0, v0.12.0, v1.0.0):
 
 1. **Prepare the Release Branch**
 
    ```sh
-   git checkout <BRANCH NAME>
-   git pull origin <BRANCH NAME>
+   git checkout main
+   git pull upstream main
+   ```
+
+2. **Create Release Branch**
+
+   Create the release branch:
+
+   ```sh
+   git checkout -b release-<MAJOR>.<MINOR>
+   git push upstream release-<MAJOR>.<MINOR>
    ```
 
-2. **Create and Push the Tag**
+3. **Create and Push the Tag**
 
    ```sh
    git tag -a <NEW VERSION> -m '<NEW VERSION>'
-   git push origin <NEW VERSION>
+   git push upstream <NEW VERSION>
+   ```
+
+4. **Publishing**
+   - GoReleaser will automatically create a new release
+   - Review and edit the release at: [GitHub Releases](https://github.com/project-copacetic/copacetic/releases)
+   - Review the respective copa-action image at: [GitHub Container Registry](https://github.com/project-copacetic/copacetic/pkgs/container/copa-action)
+   - Review the respective copa-extension image at: [GitHub Container Registry](https://github.com/project-copacetic/copacetic/pkgs/container/copa-extension)
+
+### Patch Release Process
+
+For patch releases (e.g., v<MAJOR>.<MINOR>.<PATCH>) that contain bug fixes or security patches:
```

Comment:
I think the CI failure is because the MDX compiler interprets the version template here as html. If you encase it in backticks it should resolve the build issue, i.e.:
```
(e.g., `v<MAJOR>.<MINOR>.<PATCH>`)
```